rosdep/python.yaml

```diff
@@ -1750,6 +1750,13 @@ python-flask-cors-pip:
   ubuntu:
     pip:
       packages: [flask-cors]
+python-flask-jwt-extended-pip: &migrate_eol_2025_04_30_python3_flask_jwt_extended_pip
```

Comment:
We don't need to do the migration logic for a new key using the old names. That's only needed to avoid breaking existing keys.

Reply:
done!